AnnoyancesFilter/sections/popups.txt

```diff
@@ -116,6 +116,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+fistia.proj.tokyo##.v-alert
```

Comment:
This rule has some false positive:

<details>
<summary>Screenshot</summary>

![fistia](https://user-images.githubusercontent.com/58900598/151572564-4c181c9c-b9d3-4a32-82e2-1c718a176003.png)

</details>

Please change to `fistia.proj.tokyo#?#.v-alert.info a.white--text[href="/about"]:upward(.info)` and move to annoyances.txt with a comment of this PR link.

Reply:
done